src/routes/console/project-[project]/auth/security/updatePasswordDictionary.svelte

```diff
@@ -6,10 +6,15 @@
     import { Button, Form, FormList, InputNumber, InputSwitch } from '$lib/elements/forms';
```

Comment:
We can also remove the `InputNumber` import

Reply:
updated both